packages/wrangler/src/cli.ts

```diff
@@ -28,5 +30,26 @@ export { unstable_dev, unstable_pages, unstable_DevEnv };
 export type { UnstableDevWorker, UnstableDevOptions };
 
 export * from "./api/integrations";
+
+// Export internal APIs required by the Vitest integration as `unstable_`
 export { default as unstable_splitSqlQuery } from "./d1/splitter";
 export { startWorkerRegistryServer as unstable_startWorkerRegistryServer } from "./dev-registry";
+
+// `miniflare-cli/assets` dynamically imports`@cloudflare/pages-shared/environment-polyfills`.
+// `@cloudflare/pages-shared/environment-polyfills/types.ts` defines `global`
+// augmentations that pollute the `import`-site's typing environment.
+//
+// We `require` instead of `import`ing here to avoid polluting the main
+// `wrangler` TypeScript project with the `global` augmentations. This
+// relies on the fact that `require` is untyped.
+export interface UnstableASSETSBindingsOptions {
+	log: Logger;
+	proxyPort?: number;
+	directory?: string;
+}
+const generateASSETSBinding: (
+	opts: UnstableASSETSBindingsOptions
+) => (request: Request) => Promise<Response> =
+	// eslint-disable-next-line @typescript-eslint/no-var-requires
+	require("./miniflare-cli/assets").default;
+export { generateASSETSBinding as unstable_generateASSETSBinding };
```

Comment:
This is a concern. I'm not sure of a better way of doing this right now but we likely want to find one. I guess we'd need to decouple the `@cloudflare/pages-shared/environment-polyfills/types.ts` `global` augmentations from this particular function.

Not a blocker as it's marked `unstable_` but highlighting.